lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -417,3 +418,15 @@ def test_database_calls(self, batch_size):
                     batch_size=batch_size,
                     offset=6,
                 )
+
+    @ddt.data(*xrange(1, 12, 3))
+    def test_compute_all_grades_for_course(self, batch_size):
+        """
+        Ensures that known errors are not logged before a retry.
+        """
+        self.set_up_course()
+        result = compute_all_grades_for_course.delay(
+            course_key=six.text_type(self.course.id),
+            batch_size=batch_size,
+        )
+        self.assertTrue(result.successful)
```

Comment:
This test would be more powerful if it made an assertion about the call to `compute_grades_for_course_v2()`.